src/components/DailyConfigComponent.tsx

```diff
@@ -18,41 +19,51 @@ import localStorageSet from "roamjs-components/util/localStorageSet";
 import nanoid from "nanoid";
 
 const DailyConfig = () => {
-  // TODO refresh config so debug panel shows up to date info
   const config = useMemo(getDailyConfig, []);
-  const [disabled, setDisabled] = useState(!config.enabled);
+  const [localConfig, setLocalConfig] = useState(config);
+  const {
+    enabled,
+    device,
+    time: setRunTime,
+    "workflow name": workflowName,
+    "last-run": lastRun,
+    "next-run": configNextRun,
+  } = localConfig;
   const [onlyOnThisDevice, setOnlyOnThisDevice] = useState(
-    !!config.device && config.device === localStorageGet("device")
+    !!device && device === localStorageGet("device")
   );
-  const [workflowName, setWorkflowName] = useState(config["workflow name"]);
+
   const defaultTime = useMemo(() => {
     const date = new Date();
-    if (config && config["time"]) {
-      const [h, m] = config["time"].split(":").map(Number);
+    if (localConfig && setRunTime) {
+      const [h, m] = setRunTime.split(":").map(Number);
       date.setHours(h);
       date.setMinutes(m);
     } else {
       date.setHours(0);
       date.setMinutes(0);
     }
     return date;
-  }, [config]);
-  const lastRun = config?.["last-run"];
+  }, [localConfig]);
+  const [localTime, setLocalTime] = useState(defaultTime);
```

Comment:
Id centralize with `localConfig.time`

Reply:
changed to `timePicker` to reflect that it's used in that component.  I think this is required unless we call setDailyConfig on each HH/MM change.